packages/web-react/src/components/Accordion/demo/Accordion.tsx

```diff
@@ -30,15 +30,15 @@ export const content = (
 );
 
 const Story = () => {
-  const [openState, setOpenState] = React.useState<AccordionOpenStateType>('AccordionItemExample1');
+  const [openState, setOpenState] = React.useState<AccordionOpenStateType>('AccordionItemExample-1');
```

Comment:
```suggestion
  const [openState, setOpenState] = React.useState<AccordionOpenStateType>('AccordionItemExample-1');
```
```suggestion
  const [openState, setOpenState] = useState<AccordionOpenStateType>('AccordionItemExample-1');
```

Reply:
fixed, thank you

Reply:
We should unify this everywhere but in another PR.